src/server/actions/availability/save/action.ts

```
@@ -19,19 +20,21 @@ export async function saveAvailability({
 }: saveAvailabilityProps) {
     try {
         const { user } = await getCurrentSession();
-        let memberId: string;
```

This is throwing errors. 

`23:13  Error: 'memberId' is never reassigned. Use 'const' instead.  prefer-const`

Also, you should probably throw an error if user doesn't exist